lib/tasks/test_pin_email_deliverability.rake

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+require 'faker'
+
+desc "Send PIN confirmation emails to test their deliverability, needs EMAILS_CSV_PATH var"
+task :test_pin_email_deliverability => [:environment] do
+  CSV.foreach(ENV['EMAILS_CSV_PATH']) do |row|
+
+    SignupConfirmationMailer.for_testing_deliverability(
+        email_address: row[0],
+        confirmation_pin: TokenMaker.contact_info_confirmation_pin,
+        confirmation_token: TokenMaker.contact_info_confirmation_code,
+        user_full_name: Faker::Name.name
+    )
```

Comment:
Something (either this task or the method) needs to call either `deliver_now` or `deliver_later` on the output of `mail`.

Reply:
DOH. How did I miss that. Fixed.